webrtc.html

```diff
@@ -8901,11 +8907,16 @@ <h2>Attributes</h2>
               whether the user agent is able to transmit text asynchronously
               with script execution.) This does not include framing overhead
               incurred by the protocol, or buffering done by the operating
-              system or network hardware. This attribute's value will only
+              system or network hardware. The value of the
+              <a>[[\BufferedAmount]]</a> slot will only
               increase with each call to the <code><a data-link-for=
               "RTCDataChannel">send()</a></code> method as long as the <a>
-              [[\ReadyState]]</a> slot is <code>open</code>. However, the
-              attribute does not reset to zero once the channel closes.</p>
+              [[\ReadyState]]</a> slot is <code>open</code>; however, the
+              slot does not reset to zero once the channel closes. When the
+              <a>underlying data transport</a> sends data from its queue, the
+              user agent MUST queue a task that reduces
+              <a>[[\BufferedAmount]]</a> with the number of bytes that was
+              sent.</p>
```

Comment:
With this text I guess we leave it up to implementations to decide how often to update the slot.